pkg/encrypt/encrypt.go

```diff
@@ -0,0 +1,72 @@
+package encrypt
```

Comment:
maybe this belongs under pkg/database

Reply:
Maybe `util/encrypt`? as it is used mainly in API package.

Reply:
util/encryption ?